std/testing/asserts.ts

```diff
@@ -330,69 +331,66 @@ export function assertNotStrictEquals(
 }
 
 /**
- * Make an assertion that actual contains expected. If not
- * then thrown.
- */
-export function assertStringContains(
-  actual: string,
-  expected: string,
-  msg?: string,
-): void {
-  if (!actual.includes(expected)) {
-    if (!msg) {
-      msg = `actual: "${actual}" expected to contain: "${expected}"`;
-    }
-    throw new AssertionError(msg);
-  }
-}
-
-/**
- * Make an assertion that `actual` contains the `expected` values.
+ * Make an assertion that `actual` includes the `expected` values.
  * If not then an error will be thrown.
  *
  * Type parameter can be specified to ensure values under comparison have the same type.
  * For example:
  *```ts
- *assertArrayContains<number>([1, 2], [2])
+ *assertIncludes<number>([1, 2], [2])
  *```
  */
-export function assertArrayContains(
+export function assertIncludes(
+  actual: string,
+  expected: string,
+  msg?: string,
+): void;
+export function assertIncludes(
   actual: ArrayLike<unknown>,
   expected: ArrayLike<unknown>,
   msg?: string,
 ): void;
-export function assertArrayContains<T>(
+export function assertIncludes<T>(
   actual: ArrayLike<T>,
   expected: ArrayLike<T>,
   msg?: string,
 ): void;
-export function assertArrayContains(
+export function assertIncludes(
   actual: ArrayLike<unknown>,
   expected: ArrayLike<unknown>,
   msg?: string,
 ): void {
-  const missing: unknown[] = [];
-  for (let i = 0; i < expected.length; i++) {
-    let found = false;
-    for (let j = 0; j < actual.length; j++) {
-      if (equal(expected[i], actual[j])) {
-        found = true;
-        break;
+  if (typeof actual === "string" && typeof expected === "string") {
```

Comment:
Surprising that you're special casing here, behavior should be the same across all types.